new/LIT7103PrayOTprophets.xml

```diff
@@ -0,0 +1,112 @@
+            <creation/>
+            <abstract>
+                <p>Collection of prayers of the prophets of the Old Testament, each of which are introduced with 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሐ፡</foreign> (engl. "When he cried") or 
+                    <foreign xml:lang="gez">ወሶበ፡ ጸርሑ፡</foreign> (engl. "When they cried") and is often continued with 
+                    <foreign xml:lang="gez">ስመከ፡</foreign> (engl. "your name"), possibly related to 
+                    <ref type="work" corresp="LIT1828Mahale"/>, <ref type="work" corresp="LIT2277salotu"/>,
+                    <ref type="work" corresp="LIT2265salota"/> or <ref type="work" corresp="LIT2250salota"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianContent"/>
+                    <term key="OldTestament"/>
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-10-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <bibl><ptr target="bm:Strelcyn1949PriereFalacha"/><citedRange unit="page">64-75</citedRange></bibl>
+            </div>
+            <div type="edition">
+                <note>Incipits and order of the prayers are taken from <ref type="mss" corresp="BLorient12859"/></note> 
+                <div type="textpart" xml:id="Moses">
+                    <label>Prayer attributed to Moses</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ሙሴ፡ ገበርከ፡ እንዘ፡ ይብል፡ ጋዲን፡ ፈሊዖ፡ <gap reason="ellipsis"/></q>
+                </div>
+                <div type="textpart" xml:id="Joshua">
+                    <label>Prayer attributed to Joshua</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ኢያሱ፡ ወልደ፡ ነዌ፡ ውስተ፡ ሀገረ፡ ገባዖን፡ እንዘ፡ ይብል፡ ደርታን፡</q>
+                </div>
+                <div type="textpart" xml:id="Elijah">
+                    <label>Prayer attributed to Elijah</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ኤልያስ፡ ስመከ፡ እንዘ፡ ይብል፡ አቅርያኖስ፡</q>
+                </div>
+                <div type="textpart" xml:id="Daniel">
+                    <label>Prayer attributed to Daniel</label>
+                    <q xml:lang="gez">ወሶበ፡ <supplied reason="omitted" resp="PRS8999Strelcyn">ጸርሐ፡ </supplied>ዳንኤል፡ ስመከ፡ እንዘ፡ ይብል፡ 
+                        ዝራኤል፡</q>
+                </div>
+                <div type="textpart" xml:id="Manasseh">
+                    <label>Prayer attributed to Manasseh</label>
+                    <q xml:lang="gez">ወሶበ፡ ጸርሐ፡ ምናሴ፡ ስመከ፡ እንዘ፡ ይብል፡ ቀለኤል፡</q>
+                </div>
+                <div type="textpart" xml:id="ThreeChildren">
```

Comment:
Book of Canticles contains three Songs LIT2250salota, LIT2271salota, LIT1566hababa (all from the Book of Daniel)